test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -42,6 +42,12 @@ Describe "PSReadLine" -tags "CI" {
         (Get-PSReadLineKeyHandler | where { $_.Key -ceq "Alt+B" }).Function | Should Be SelectBackwardWord
     }
 
+    It "Should ignore case when using Function binding" {
+        Set-PSReadlineOption -EditMode Windows
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function yank # NOTE: lower case function name
+        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be Yank
```

Comment:
FYI, the BeExactly part is not neccessary. As long as Function is not empty, the test has passed.